spec/depth-enabled-photography.html

```diff
@@ -2160,121 +2160,417 @@
+                },
+                function(error) { console.log(error.message); });
+
+            photo.queryDeviceVendorInfo().then(
+                function(vendorInfo) {
+                  console.log('model: ' + venterInfo.model + ', manufacturer: ' + venterInfo.manufacturer +
+                      ', notes: ' + venterInfo.notes);
+                },
+                function(error) { console.log(error.message); });
+
+            photo.queryCameraVendorInfo(0).then(
+                function(vendorInfo) {
+                  console.log('model: ' + venterInfo.model + ', manufacturer: ' + venterInfo.manufacturer +
+                      ', notes: ' + venterInfo.notes);
+                },
+                function(error) { console.log(error.message); });
+
+            photo.queryCameraPerspectiveModel(0).then(
+                function(perspectiveModel) {
+                  console.log('model: ' + perspectiveModel.model +
+                      ', focalLength: (' + perspectiveModel.focalLength.x + ', ' +
+                                           perspectiveModel.focalLength.y + ')' +
+                      ', principalPoint: (' + perspectiveModel.principalPoint.x + ', ' +
+                                              perspectiveModel.principalPoint.y + ')' +
+                      ', skew: ' + perspectiveModel.skew +
+                      ', radialDistortion: [' + perspectiveModel.radialDistortion.k1 + ', ' +
+                                                perspectiveModel.radialDistortion.k2 + ', ' +
+                                                perspectiveModel.radialDistortion.k3 + ']' +
+                      ', tangentialDistortion: [' + perspectiveModel.tangentialDistortion.p1 + ', ' +
+                                                    perspectiveModel.tangentialDistortion.p2 + ']');
+                },
+                function(error) { console.log(error.message); });
+
+            photo.queryCameraPose(0).then(
+                function(pose) {
+                  console.log('transition: (' + pose.transition.x + ', ' +
+                                                pose.transition.y + ', ' +
+                                                pose.transition.z + ')' +
+                      ', rotation: (' + pose.rotation.rotationAngle + ', ' +
+                                        pose.rotation.rotationAxisX + ', ' +
+                                        pose.rotation.rotationAxisY + ', ' +
+                                        pose.rotation.rotationAxisZ + ')');
+                },
+                function(error) { console.log(error.message); });
+
+            photo.queryImage().then(
+                function(image) {
+                  // render the color image.
+                },
+                function(error) { console.log(error.message); });
+
+            photo.queryDepth().then(
+                function(image) {
+                  // render the depth image.
```

Comment:
Same comments of color image. It would be more intuitive to add some comments of depth image content.